### DISCUSSION TITLE
Frage zu QR Code auf Mobiles Gerät übertragen

### DISCUSSION THREAD

**Discussion by Sunbeam-ops** [2fast-team/2fast#189] [Q&A] [answered]
Hallo,

ich hab 2ast vorhin installiert und gekauft, weil ich genau das gesucht habe. 
Allerdings wird mir auf einem Mobilen Gerät "iphone" angezeigt das der QR Code ungültig ist wenn ich den Code mit der Kamera einlese ?
Ich hab das mit dem Microsoft Authenticator und eine andere App versucht. Beides zeigt mir an das der Code ungültig ist ?
Ist das noch in Entwicklung ?

**Comment by jp-weber** [comment]
Hallo, vielen Dank für die Unterstützung!

Das sollte eigentlich ohne Probleme funktionieren. Um welchen Dienst handelt es sich? Dann könnte ich das einmal nachstellen.

**Comment by Sunbeam-ops** [comment]
Hallo Herr Weber,

Vielen Dank für die schnelle Antwort.
Es geht um die Webseite / Spiel Spaceinvasion ?
https://board.bitmeup.com/
Bin grad dabei alles neu einzurichten da ein Import aus einer anderen App scheinbar nicht funktioniert ? Oder ich hab da was übersehen ?
In der 2fast App hab ich oben rechts auf die 3 Punkte geklickt und auf QR Code anzeigen geklickt und da bekomm ich die Meldung QR Code ungültig ? beziehungsweise auf der Mobilen App 

Edit: es betrifft tatsächlich nur diese Seite https://board.bitmeup.com/ bisher. Alle anderen Webseiten konnte ich den Code einscannen.
Ist jetzt kein Beinbruch und kann damit Leben, da ich das Spiel für Unterwegs eh nicht nutze.
Super Sache anstatt jedesmal das Handy in die Hand zu nehmen :)) 👍

**Reply by jp-weber** [reply, marked as answer]
Hallo,

ich habe ein Konto bei bitmeup erstellt und die Ursache des Problems gefunden. Der Tag des Kontos (#) hat bei der Erstellung des QR Codes Probleme bereitet, dies ist nun behoben und wird in dem nächsten Update (1.3.4) verteilt. Als Workaround kann bis zu dem Update, in der 2fast App das Konto für bitmeup bearbeitet werden und der # z.B. durch einen Unterstrich ersetzt werden.

**Comment by Sunbeam-ops** [comment]
Guten Morgen,

alles gut,  dann weiß ich bescheid. Vielen Dank für die Info.

Gruß
Mike
